static/app/components/modals/dashboardWidgetQuerySelectorModal.tsx

```diff
@@ -107,7 +107,6 @@ const OpenInDiscoverButton = styled(Button)`
 
 const Container = styled('div')`
   border: 1px solid ${p => p.theme.border};
-  box-shadow: inset ${p => p.theme.dropShadowLight};
```

Comment:
<img width="490" alt="Screen Shot 2021-09-28 at 4 02 11 PM" src="https://user-images.githubusercontent.com/44172267/135177095-91ceb08a-234c-46b3-8d54-99315d3a2d8d.png">

(see: the input fields)

Reply:
👍